includes/posting.inc.php

```diff
@@ -130,6 +130,8 @@
 	$action = 'unlock_thread';
 if (isset($_GET['edit']))
 	$action = 'edit';
+if (isset($_GET['report_entry']))
```

Comment:
More a general question/comment (and maybe not the only place in the script): Isn't better to use an `elseif` instead of multiple `if` statements?

Reply:
With the multiple `if`s it is possible to overwrite a first found GET-parameter with the second/further one. When we would use `elseif` instead the first parameter would get the go and every further parameter would get ignored. Both ways are reasonable and have their advantages and disadvantages. And that (dis)-advantages depends also on disciplined we are with creating the parameters on the sending side. I am undecided if we should remain with `if` or change to `elseif` because this can lead to necessary checks and changes on other places of the code.

Reply:
No, I disagree. If the parameter can be overwritten, we just have to change the order, i.e.

```
if #1 
  do1
if #2
  do2
if #3
  do3
```

is the same as

```
if #3
  do3
elseif #2 
  do2
else
  do1
```

Reply:
Ok, if you reverse the order you are right. But if we change it here, we have to change it everywhere for consistency. On the other hand, this should *IMHO* not be a part of *this* pull request.

Reply:
Yes, it was a more generally question!